source/draw/RasterMonochrome.ooc

```diff
@@ -203,4 +203,25 @@ RasterMonochrome: class extends RasterPacked {
 		}
 		((this buffer pointer + y * this stride) as ColorMonochrome* + x)@ = value
 	}
+	getLine: func (y: Int) -> FloatVectorList {
+		result := FloatVectorList new()
+		version(safe) {
+			if (y > this size height || y < 0)
+				raise("Accessing RasterMonochrome index out of range in getLine")
+		}
+		for (x in 0..(this size width))
+				result add(this buffer pointer[y * this stride + x] as Float)
+		result
+	}
+
+	getColon: func (x: Int) -> FloatVectorList {
```

Comment:
"colon" is "tjocktarm", among other things. I think you mean "column".

But, some consistency would be in order. If you have a getColumn function, the other one should be getRow, not getLine.